lib/assert.js

```diff
@@ -131,6 +177,7 @@ assert.AssertionError = AssertionError;
 function ok(...args) {
   innerOk(ok, args.length, ...args);
 }
+Assert.prototype.ok = ok;
```

Comment:
That does mean that `new Assert().ok.strictEqual` would be a thing, which doesn't feel as clean as it could be, we could consider change that if that's not too complex

Reply:
Good point, I hadn't thought of that case, but we still need the individual's `function ok` to be exported, so I'll just implement the same function to `Assert`